lib/utils.js

```diff
@@ -64,12 +64,21 @@ var utils = module.exports = {
         simpleGit(cwd).revparse([ '--abbrev-ref', 'HEAD' ], callback)
       })
         .then(function (branch) {
-          return branch.toString().trim()
+          if (branch) {
+            return branch.toString().trim()
+          }
+          return branch
```

Comment:
You're right. This will always return something. I was thinking there might be a scenario where this wouldn't return, but it doesn't really happen. I added this so that if there wasn't any branch, the error would get triggered below (but this doesn't happen).